deploy/contents/k8s/cp-search/cp-search-srv-dpl.yaml

```diff
@@ -26,6 +26,9 @@ spec:
           command: ["/init"]
           ports:
             - containerPort: 8080
+          env:
+            - name: CP_SEARCH_DISABLE_NFS_OBSERVER_SYNC
+              value: "true"
```

Comment:
1. Since we are changing this deployment file anyway, let's add the following entry:
```yaml
- name: CP_SEARCH_DISABLE_NFS_FILE
  value: "false"
```
It is a default value, that might be handy to have in place, so we can easily change the deployment file, when the environment should have a separate NFS synchronizer.

2. Maybe we should add `LANG` env var as well

Reply:
Both variables were added to the deployment file.